Common/Kernel/Middlewares/CheckTokenMiddleware.cs

```diff
@@ -25,7 +25,7 @@ public CheckTokenMiddleware(RequestDelegate next, [FromServices] IBus bus)
         public async Task InvokeAsync(HttpContext context)
         {
             if (string.Equals(context.Request.Method, RestRequestType.POST.ToString(), StringComparison.OrdinalIgnoreCase)
-                && string.Equals(context.Request.Path, "/users/create", StringComparison.OrdinalIgnoreCase))
+                && string.Equals(context.Request.Path, "/user/create", StringComparison.OrdinalIgnoreCase))

```

Comment:
Revert this back, controller should have User**s**Controller name

Reply:
It was an accident
Now fixed